README.md

```diff
@@ -31,6 +31,10 @@ A curated list of awesome Dash (plotly) resources
 - [Visualize Earthquakes with Plotly Dash](https://www.giacomodebidda.com/visualize-earthquakes-with-plotly-dash/) - Environmental scan of alternatives to Dash followed with a tutorial.
 - [ARGO Labs - Plotly Dash Tutorial (Video)](https://www.youtube.com/watch?v=yfWJXkySfe0) - Detailed introduction to creating interactive dashboards.
 - [Data Visualization GUIs with Dash and Python (Video playlist)](https://www.youtube.com/watch?v=J_Cy_QjG6NE&list=PLQVvvaa0QuDfsGImWNt1eUEveHOepkjqt) - Five-part series exploring Dash features.
+- [Interactive Image Processing with Dash-Canvas](https://www.youtube.com/watch?v=LKXSFBB5ccI) - Official introduction video to the Dash Canvas components.
+- [Dash Cytoscape Component](https://www.youtube.com/watch?v=snXcIsCMQgk) - Official introduction video to the Dash Cytoscape components.
+- [An introduction to Dash DataTable](https://www.youtube.com/watch?v=dueejcyrYh8) - Official introduction video to the Dash DataTable components.
+- [Introducing the JupyterLab Chart Editor](https://www.youtube.com/watch?v=zR7G2tNVo1Q) - Official introduction video to the JupyterLab Chart Editor.
```

Comment:
Thought I think JupyterLab is fantastic, I don't think it is strictly related to dash. I'd be inclined to remove this from the commit. Thoughts?

Reply:
Sidenote: Thanks for all the contributions and sorry I didn't see them earlier. Feel free to request me as a reviewer in future so I get a notification to merge them more promptly!

Reply:
You are right. However I experienced JupyterLab as a super helpful development tool for Dash apps. Single parts of the UI can be rapid prototyped with JupterLab in a very efficient manner. Anyway: I can track the JupyterLab video in my notes and remove it from the PR. Just let me know.

Don't worry, PRs are patient 😉. Thx, I'll do so.

Reply:
Quick and dirty fixed.